src/main/java/com/JellyWorks/controller/Controller.java

```diff
@@ -0,0 +1,39 @@
+package com.JellyWorks.controller;
+
+import java.net.URISyntaxException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import com.JellyWorks.storage.Local.LocalInputInvoker;
+import com.JellyWorks.storage.Local.LocalInputProcessor;
+import com.JellyWorks.storage.s3.S3Processor;
+
+@Component
+@RestController
+@RequestMapping("/")
+public class Controller {
+
+	LocalInputProcessor localProcessing;
+	@Autowired
+	S3Processor s3processor;
+	
+	@RequestMapping("/home")
+	public String index() {
+		return "Greetings from Spring Boot!";
+	}
+	
+	@RequestMapping(value="/readFile", method=RequestMethod.GET, params="file")
```

Comment:
Eventually let us not expose language exceptions to the end user. Instead we will have meaningful error codes and descriptions.